docs/en/solutions/apply_catboost_model.md

```diff
@@ -0,0 +1,204 @@
+# Applying a CatBoost model in ClickHouse {#applying-catboost-model-in-clickhouse}
+
+[CatBoost](https://catboost.ai) — is a free and open-source gradient boosting library for machine learning.
+
+To apply a CatBoost model in ClickHouse:
+
+1. [Create a table](#create-table).
+2. [Insert the data to the table](#insert-the-data-to-the-table).
+3. [Configure the model](#configure-the-model).
+4. [Run the model inference from SQL](#run-the-model-inference).
+
+For more information about training CatBoost models, see [Training and applying models](https://catboost.ai/docs/features/training.html#training).
+
+## Before you start {#before-you-start}
+
+If you don't have the [Docker](https://docs.docker.com/install/) yet, install it.
+
+> **Note:** [Docker](https://www.docker.com) uses containers to create virtual environments that isolate a CatBoost and ClickHouse installation from the rest of the system. CatBoost and ClickHouse programs are run within this virtual environment.
+
+Before applying a CatBoost model:
+
+**1.** Pull the [Docker image](https://hub.docker.com/r/yandex/tutorial-catboost-clickhouse) from the registry:
+
+```bash
+$ docker pull yandex/tutorial-catboost-clickhouse
+```
+
+This Docker image contains everything you need to run an application: code, runtime, libraries, environment variables, and configuration files.
+
+**3.** Make sure the Docker image has been pulled:
+
+```bash
+$ docker image ls
+REPOSITORY                            TAG                 IMAGE ID            CREATED             SIZE
+yandex/tutorial-catboost-clickhouse   latest              3e5ad9fae997        19 months ago       1.58GB
+```
+
+**2.** Start the Docker-configured image:
+
+```bash
+$ docker run -it -p 8888:8888 yandex/tutorial-catboost-clickhouse
+```
+
+> **Note:** Example running a Jupyter Notebook with this manual materials to [http://localhost:8888](http://localhost:8888).
```

Comment:
* incorrect markdown for note block
* btw have you checked the contents of that Jupyter Notebook? it might have some documentation-like comments that need editing

Reply:
тогда видимо надо будет проапдейтить их отредактированной версией после мерджа PR